engineering/documents/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing Guidelines
+
+source{d} go-git project is [Apache 2.0 licensed](LICENSE) and accept
+contributions via GitHub pull requests.  This document outlines some of the
+conventions on development workflow, commit message formatting, contact points
+and other resources to make it easier to get your contribution accepted.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+## Support Channels
+
+Whether you are a user or contributor, official support channels include:
+
+- GitHub [issues](https://github.com/src-d/go-git/issues/new)
+- Slack: #go-git room in the [source{d} Slack](http://sourced.tech/)
```

Comment:
I'd suggest point people to the issues rather than creating new issues (makes people more likely to search for the problem first, see current issues, etc). 
I.e.: `- GitHub [issues](https://github.com/src-d/go-git/issues)*.`
> - GitHub [issues](https://github.com/src-d/go-git/issues)*.

Also we can point people directly to the related community channel (or #general in case there's none) but let them know they need to sign up first (unfortunately we don't have https://github.com/src-d/backlog/issues/1077 yet to link to it). 
I.e.: `- Slack [#go-git channel](https://sourced-community.slack.com/messages/go-git), after [joining the community](https://sourced.tech/) at source{d}.`
> - Slack [#go-git channel](https://sourced-community.slack.com/messages/go-git), after [joining the community](https://sourced.tech/) at source{d}.

Reply:
More of a question: worth to link to documentation here?

Reply:
witch documentation?